tests/Doxyfile

```diff
@@ -673,7 +673,7 @@ INPUT_ENCODING         = UTF-8
 # *.hxx *.hpp *.h++ *.idl *.odl *.cs *.php *.php3 *.inc *.m *.mm *.dox *.py
 # *.f90 *.f *.for *.vhd *.vhdl
 
-FILE_PATTERNS          = *.cpp *.c *.h *.dox
+FILE_PATTERNS          = *.h *.dox
```

Comment:
Wouldn't it be better to exclude the bits we don't care about? Or is this to force stuff to be documented in the .h?